package.json

```diff
@@ -7,15 +7,15 @@
     "test": "test",
     "authlocalinvoke": "sam local invoke IPAuthorizerLambda --event ip-authorizer/event.json --env-vars ip-authorizer/env.json",
     "backlocalinvoke": "sam local invoke BackendLambda --event backend-lambda/event.json --env-vars backend-lambda/env.json",
-    "apilocalinvoke": "sam local start-api --env-vars backend-lambda/env.json"
+    "apilocalinvoke": "sam local start-api --env-vars backend-lambda/env.json",
+    "deploysam": "sam build && sam deploy --stack-name im-sam --resolve-s3 --region us-east-1 --confirm-changeset",
+    "deployapp": "cd react-cors-spa && npm run build && aws s3 sync build/ s3://im.react.app --delete"
```

Comment:
Update this to clear out the bucket before we stick new files in to make sure we don't get artifacts laying around.

Reply:
for deploysam?

deployapp will delete any resources that don't match the local directories/files that are being pushed

Reply:
Oh sick my b!